x/harvest/keeper/repay.go

```diff
@@ -0,0 +1,56 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/kava-labs/kava/x/harvest/types"
+)
+
+// Repay borrowed funds
+func (k Keeper) Repay(ctx sdk.Context, sender sdk.AccAddress, coins sdk.Coins) error {
+	// Validate requested repay
```

Comment:
Need to call `k.SyncBorrowInterest` so that repayment includes interest.

Reply:
https://github.com/Kava-Labs/kava/pull/725/commits/2723a8e2dbd9722624a007417cd6e30421eb2a91